### PR TITLE
use mvnd in Spark integration test jobs

### PR DESCRIPTION
**Pull request by bowenliang123**
### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by bowenliang123** [comment]
```
[INFO] [stdout] PySparkTests:
[INFO] [stdout] - pyspark support *** FAILED ***
[INFO] [stdout]   org.apache.kyuubi.jdbc.hive.KyuubiSQLException: Failed to init python environment, fall back to SQL mode: SPARK_HOME not found!
[INFO] [stdout]   at org.apache.kyuubi.jdbc.hive.Utils.verifySuccess(Utils.java:88)
[INFO] [stdout]   at org.apache.kyuubi.jdbc.hive.Utils.verifySuccessWithInfo(Utils.java:74)
[INFO] [stdout]   at org.apache.kyuubi.jdbc.hive.KyuubiStatement.runAsyncOnServer(KyuubiStatement.java:325)
[INFO] [stdout]   at org.apache.kyuubi.jdbc.hive.KyuubiStatement.executeWithConfOverlay(KyuubiStatement.java:197)
[INFO] [stdout]   at org.apache.kyuubi.jdbc.hive.KyuubiStatement.execute(KyuubiStatement.java:192)
[INFO] [stdout]   at org.apache.kyuubi.jdbc.hive.KyuubiStatement.executeQuery(KyuubiStatement.java:450)
[INFO] [stdout]   at org.apache.kyuubi.engine.spark.operation.PySparkTests.$anonfun$runPySparkTest$1(PySparkTests.scala:139)
[INFO] [stdout]   at org.apache.kyuubi.engine.spark.operation.PySparkTests.$anonfun$runPySparkTest$1$adapted(PySparkTests.scala:137)
[INFO] [stdout]   at org.apache.kyuubi.operation.JDBCTestHelper.$anonfun$withMultipleConnectionJdbcStatement$3(JDBCTestHelper.scala:60)
[INFO] [stdout]   at org.apache.kyuubi.operation.JDBCTestHelper.$anonfun$withMultipleConnectionJdbcStatement$3$adapted(JDBCTestHelper.scala:60)
[INFO] [stdout]   ...
[INFO] [stdout]   Cause: java.lang.RuntimeException: org.apache.kyuubi.KyuubiSQLException:Failed to init python environment, fall back to SQL mode: SPARK_HOME not found!
[INFO] [stdout]   at org.apache.kyuubi.KyuubiSQLException$.apply(KyuubiSQLException.scala:69)
[INFO] [stdout]   at org.apache.kyuubi.engine.spark.operation.SparkSQLOperationManager.newExecuteStatementOperation(SparkSQLOperationManager.scala:102)
[INFO] [stdout]   at org.apache.kyuubi.session.AbstractSession.$anonfun$executeStatement$1(AbstractSession.scala:130)
[INFO] [stdout]   at org.apache.kyuubi.session.AbstractSession.withAcquireRelease(AbstractSession.scala:82)
[INFO] [stdout]   at org.apache.kyuubi.session.AbstractSession.executeStatement(AbstractSession.scala:128)
[INFO] [stdout]   at org.apache.kyuubi.service.AbstractBackendService.executeStatement(AbstractBackendService.scala:66)
[INFO] [stdout]   at org.apache.kyuubi.service.TFrontendService.ExecuteStatement(TFrontendService.scala:252)
[INFO] [stdout]   at org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement.getResult(TCLIService.java:1557)
[INFO] [stdout]   at org.apache.hive.service.rpc.thrift.TCLIService$Processor$ExecuteStatement.getResult(TCLIService.java:1542)
[INFO] [stdout]   at org.apache.thrift.ProcessFunction.process(ProcessFunction.java:39)
[INFO] [stdout]   ...
[INFO] [stdout]   Cause: java.lang.IllegalStateException: SPARK_HOME not found!
[INFO] [stdout]   at org.apache.kyuubi.engine.spark.operation.ExecutePython$.$anonfun$defaultSparkHome$6(ExecutePython.scala:342)
[INFO] [stdout]   at scala.Option.getOrElse(Option.scala:189)
```

File: .github/workflows/master.yml [hunk 1]
```diff
@@ -77,14 +77,18 @@ jobs:
...
         with:
           python-version: '3.9'
       - name: Build and test Kyuubi and Spark with maven w/o linters
         run: |
           TEST_MODULES="dev/kyuubi-codecov"
-          ./build/mvn clean install ${MVN_OPT} -pl ${TEST_MODULES} -am \
+          ./build/mvnd clean install ${MVN_OPT} -pl ${TEST_MODULES} -am -DskipTests \
+          -Pspark-${{ matrix.spark }} ${{ matrix.spark-archive }} ${{ matrix.exclude-tags }}
+          ./build/mvnd test ${MVN_OPT} -pl ${TEST_MODULES} -am \
           -Pspark-${{ matrix.spark }} ${{ matrix.spark-archive }} ${{ matrix.exclude-tags }}
```
Comment (zwangsheng) [inline .github/workflows/master.yml:92]: Duplicate line

Comment (bowenliang123) [reply .github/workflows/master.yml:92]: Nope.

**Comment by bowenliang123** [comment]
Closing this PR, as the project dropped usage of `mvnd`.

**Comment by bowenliang123** [comment]
Closing this PR, as the project dropped usage of `mvnd`.

**Pull request closed without merging**